Index: web/js/app/controller/Dbgl.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- web/js/app/controller/Dbgl.js	(revision 2afd8c9b803522f313039ea4a68395b31e7ee4e6)
+++ web/js/app/controller/Dbgl.js	(revision )
@@ -274,6 +274,7 @@
     },
     updateformbyurl:function(btn,url,processtype){
         var form=btn.up('form');
+        var me=this;
 
         var head_cl=this.application.getController("Header");
         var s_arr=head_cl.signaturepicarr;
@@ -319,8 +320,12 @@
             params['processstatustype']=processtype;
         }
 
-        var successFunc = function (form, action) {
-            Ext.Msg.alert("提示信息", "更新数据成功");
+        var successFunc = function (myform, action) {
+            Ext.Msg.alert("提示信息", "操作成功");
+            var hc=me.application.getController("Header");
+            hc.closetab(form.id);
+            var grid=form.objdata.grid;
+            grid.getStore().load();
 
         };
         var failFunc = function (form, action) {
@@ -349,11 +354,11 @@
             isapproval: ajaxform.getForm().getValues().approvalresult==approvalresult.yes,
             approvalname:'街道/乡镇审核'
         };
-        var successFunc = function (form, action) {
+        var successFunc = function (myform, action) {
             btn.up('window').close();
             Ext.Msg.alert("提示信息", "审核成功");
             var hc=me.application.getController("Header");
-            hc.closetab("tabdbglbusinessalterform");
+            hc.closetab(form.id);
             grid.getStore().load();
 
         };
